pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [tool.poetry]
-name = "d-fast_morphological_impact"
+name = "dfastmi"
 version = "2.0.8"
```

Comment:
should this also be ""3.0.0"

Still little bit unclear to me why this is incorrect.

Reply:
Not sure, I can update the version, Not sure how our versioning works :)

Reply:
I am saying it because @ralph-peelen-deltares stated it in the example of discord, not sure how this would directly impact expected versioning in the software...

Reply:
discussed, working for Prisca.